.travis.yml

```diff
@@ -1,31 +1,53 @@
-language: python
-sudo: required
 dist: trusty
+sudo: false
+
+language: python
 python:
-- '2.7'
+  - '2.7'
+
+env:
+  global:
+    - GETH_URL='https://github.com/ethereum/go-ethereum/releases/download/v1.4.18/geth-linux-amd64-1.4.18-ef9265d0.tar.gz'
+    - GETH_VERSION='1.4.18'
+    - SOLC_URL='https://github.com/brainbot-com/solidity-static/releases/download/v0.4.2/solc'
+    - SOLC_VERSION='0.4.2'
```

Comment:
It's gonna be quite annoying to have to specify new versions manually, but it makes sense.
Can you please update Solidity to `0.4.4` here? https://github.com/ethereum/solidity/releases